# Messaggi del classificatore

Header header

uint8 UNLOCKED = 4
uint8 LOCKED = 5
uint8 type